tasks/main.yml

```
@@ -5,23 +5,113 @@
 
 # Examples of some tasks:
 - name: Ensure required packages are installed
-  package:
+  ansible.builtin.package:
     name: "{{ __aide_packages }}"
     state: present
     use: "{{ (__aide_is_ostree | d(false)) |
              ternary('ansible.posix.rhel_rpm_ostree', omit) }}"
+  when:
```

```suggestion
  when: aide_install | bool
```
Please change accordingly everywhere

---

done